### ISSUE TITLE
Help wanted: Font Rendering issue, seems not support Nerd Fonts?

### ISSUE THREAD

**Issue by stark-zhang** [Eugeny/tabby#5474]
I am a new user of Tabby alpha 169. when I use neovim to open C++ source code, I found the font rendering is strange if making comparison with Windows Terminal.

Configuration of Tabby(As same as windows terminal):
![image](https://user-images.githubusercontent.com/17326718/149463375-7cfe5c3b-6a86-4fc2-9891-a5e780d0e1f2.png)

Test Environment:

+ Host Operating System: Windows 10 2004 enterprise
+ Remote Operating System: Cent OS 7.9
+ Neovim: `v0.6.0`
+ Plugins: [coc.nvim](https://github.com/neoclide/coc.nvim) + [ccls](https://github.com/MaskRay/ccls) + [vim-lsp-cxx-highlighting](https://github.com/jackguo380/vim-lsp-cxx-highlight)(all latest version)
+ Windows Terminal: `1.12.3472.0`

screenshot from Windows Terminal:
![image](https://user-images.githubusercontent.com/17326718/149459531-97f1730f-081a-476d-bf6b-8eed2aebc8fe.png)

screenshot with Tabby alpha 169:
![image](https://user-images.githubusercontent.com/17326718/149459733-cafa3ccb-6be5-44cb-8eab-56f838f5fb10.png)

I think you could find that, the blur of C++ keyword, the highlight of member variable is indistinguishable, the floating window is 
pierced by the letter `r`, and the flag column diverts the position of line number column, and so on.

And the many of my issues disappeared when I used `Consolas` as the default font, except the keyword blur and indistinguishable highlight of member variable. so are these issues caused by nerd fonts(`FiraCode NF` is one of nerd fonts)?

screenshot with Tabby alpha 169, used font `Consolas`. you can find the behaviours of floating window and flag column are expected.
![image](https://user-images.githubusercontent.com/17326718/149464285-59c8b783-bc10-4e43-a227-fd90b1c50e16.png)

P.S.: new bug when run command `:version` in neovim with `FiraCode NF`:
![image](https://user-images.githubusercontent.com/17326718/149463608-daae0a36-e808-46c7-b067-2412ecf6e949.png)

**Comment by stark-zhang** [comment]
could anyone give some help or reply about this issue?

**Comment by 1JunGu** [comment]
I'm a newbie of this app. I also want to use `Fira code` font in this app, could you please tell me how to set it? Similar test environments.

**Comment by Kyllingene** [comment]
I have a similar issue with Fira Code and other Cascadia Mono offshoots, where italic characters get cut off where their character cells end.

**Comment by stark-zhang** [comment]
> I'm a newbie of this app. I also want to use `Fira code` font in this app, could you please tell me how to set it? Similar test environments.

I have given it up and been back to Windows Terminal.

**Comment by 1JunGu** [comment]
> 

Thanks for your quick reply! I also think Windows Terminal is good because of it's less memory usage and good GPU rendering.

**Comment by 1JunGu** [comment]
> > could you please tell me how to set it?
> 
> Go into `Appearance` and set `Font` to `Fira Code` after installing it on your device.

Thanks a lot. I forgot to restart `Tabby`

**Comment by Kyllingene** [comment]
I don't believe I had to restart Tabby; all I have to do is go back to the tab I was on before.

**Comment by Ultra980** [comment]
I also have some strange issues with Jetbrains Mono NF. Sometimes, the ligatures have some random character in the middle of them.

**Comment by 1JunGu** [comment]
I have no similar errors in Tabby using Mono-series Fonts, Would you update to latest version and download full-version Font (thin bold or italic) ?

在2023-02-09 15:46:10，Alex ***@***.***写道：

I also have some strange issues with Jetbrains Mono NF. Sometimes, the ligatures have some random character in the middle of them.

**Comment by carrotfarmer** [comment]
Even I have issues with JetBrains Mono NF. Especially in nvim

**Comment by carrotfarmer** [comment]
<img width="228" alt="image" src="https://user-images.githubusercontent.com/66675022/236133837-5d391cc3-8eca-4207-82a4-6f7bee184730.png">

<img width="148" alt="image" src="https://user-images.githubusercontent.com/66675022/236133925-fa93d87b-ebf8-4967-a365-415e4f738e7d.png">

<img width="174" alt="image" src="https://user-images.githubusercontent.com/66675022/236133959-2d3181d5-c5ea-416d-a94c-885e33968508.png">

**Comment by ThinkSalat** [comment]
Same issues here. Something wrong with the font renderer it seems

**Comment by jaltgen** [comment]
I am experiencing similar issuer on Windows.

![image](https://github.com/Eugeny/tabby/assets/517885/f90604d2-b429-409d-a4ba-bbcb08fa107d)
Also, it strangely highlights the letter i when i type "fish" (to run the shell)
This is running
- tmux
- Tabby 1.0.207 on Windows 11 with WSL 2.0 Ubuntu 22.04
- Fira Code Mono Nerdfont

I'll try playing around with the terminal render settings in tmux, nvim and the like, they were working with this config on my Mac and my Windows 10 / WSL 2.0,

**Comment by axel-paillaud** [comment]
Hello, same problem here with FiraCode NF and Maple Font NF (less worse with Maple Font NF). No problem with Liberation Mono.
![image](https://github.com/Eugeny/tabby/assets/85078268/7eb12d26-7771-436f-bb6a-ba4ac4707540)
